src/mapbuffer.cpp

```diff
@@ -541,7 +541,12 @@ void mapbuffer::deserialize( JsonIn &jsin )
                     int i = jsin.get_int();
                     int j = jsin.get_int();
                     // TODO: jsin should support returning an id like jsin.get_id<trap>()
-                    sm->trp[i][j] = trap_str_id( jsin.get_string() );
+                    const trap_str_id trid( jsin.get_string() );
+                    if( trid == "tr_brazier" ) {
+                        sm->frn[i][j] = furn_id( "f_brazier" );
+                    } else {
+                        sm->trp[i][j] = trid.id();
+                    }
```

Comment:
Done in https://github.com/CleverRaven/Cataclysm-DDA/pull/21321/commits/158ff80edacdb1c575279ed427d1503b2c3afa09.